### ISSUE TITLE
Still works for 2024 sai v2 versions!

### ISSUE THREAD

**Issue by LAP87** [tututu-patch/saicrack#1]
Not an issue, just wanted to let everyone know it still works, even though it's 3 years past last commit :)
Also, don't forget to inactivate anti-virus live protection, it hinders the extraction of saicrack.exe.

**Comment by NeZothking** [comment]
is it a virus?
![image](https://github.com/tututu-patch/saicrack/assets/137281412/6c4a21b5-7b72-4e44-9c4e-362a25117033)

**Comment by LAP87** [comment]
> is it a virus? ![image](https://private-user-images.githubusercontent.com/137281412/312498823-6c4a21b5-7b72-4e44-9c4e-362a25117033.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAzNDIzNzIsIm5iZiI6MTcxMDM0MjA3MiwicGF0aCI6Ii8xMzcyODE0MTIvMzEyNDk4ODIzLTZjNGEyMWI1LTdiNzItNGU0NC05YzRlLTM2MmEyNTExNzAzMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxM1QxNTAxMTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lZTc0MWZlZDIxMzc2ZGRlOGU4Y2FjYWU4ZDhmZTY3ZWQ0MDJlM2UyMTQ4YTVmZDRkY2Y0YmU2OGM2NzU4OWY0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.AwWulyJDyaZZPeOxBNw57E0dfNTbDvYIDXgVR94KQGo)

No, it's just false positive as usual with cracks, but it's only used to generate a license file then you can remove it, it doesn't install anything, the whole procedure can be read from the .py file
Also stop using chrome browser, they really do not have your best interest at heart

**Comment by Enzvioxity** [comment]
it is saying liscence broken for me
